### PR TITLE
mccp-23

### PR DESCRIPTION
**Pull request by jkusne**
add-mccp-23

### CODE REVIEW

File: MCCP/mccp-23.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+title: Disable underutilized dials II
+status: Proposed
+author: Dimitri Golecko (@dimsome), Julian Kusnetzoff (@jkusne)
+discussions-to: https://forum.mstable.org/t/rfc-disable-all-dials-except-staking-treasury-and-create-a-v2-incentives-one/894
+created: 2022-07-26
+---
+
+## Simple Summary
+
+It is proposed to disable the second batch of underutilized dials in the Emissions Controller as launched with [MIP 24](https://mips.mstable.org/MIPS/mip-24.html) in order to avoid incentivising pools that receive a substantial amount of MTA per epoch but do not contribute effectively to the usage of the mStable protocol and add value by contributing to overall revenue.
```
Comment (dimsome) [inline MCCP/mccp-23.md:12]: Please do a relative path here

File: MCCP/mccp-23.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## Abstract
+
+Dials receive votes from MTA stakers that decide the allocation of newly emitted MTA. Some participants choose to actively shift their allocation while others allocate once and allow it to be dormant. This creates a dynamic in which some of the dials receive fairly few votes from a few individuals that haven’t adjusted their weights in a long while.
+
+The first set of dials was disabled with [MCCP 22](https://mips.mstable.org/MCCP/mccp-22.html). This resulted in a much more streamlined emission process by avoiding dust MTA to be sent to illiquid pools.
```
Comment (dimsome) [inline MCCP/mccp-23.md:18]: Please do a relative path here

File: MCCP/mccp-23.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+mccp: 23
+title: Disable underutilized dials II
+status: Proposed
+author: Dimitri Golecko (@dimsome), Julian Kusnetzoff (@jkusne)
+discussions-to: https://forum.mstable.org/t/rfc-disable-all-dials-except-staking-treasury-and-create-a-v2-incentives-one/894
```
Comment (dimsome) [inline MCCP/mccp-23.md:6]: Can leave blank for now as this should point to the forum discussion when we post this MCCP, will update later

**Review by dimsome — CHANGES_REQUESTED** [review]
Some smoll things

File: MCCP/mccp-23.md (hunk 1 above)
Comment (jkusne) [reply MCCP/mccp-23.md]: done

File: MCCP/mccp-23.md (hunk 2 above)
Comment (jkusne) [reply MCCP/mccp-23.md]: done

File: MCCP/mccp-23.md (hunk 3 above)
Comment (jkusne) [reply MCCP/mccp-23.md]: done

File: MCCP/mccp-23.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+title: 
+status: 
+author: 
+discussions-to: 
+created: 
+---
+
+## Simple Summary
+
+It is proposed to disable the second batch of underutilized dials in the Emissions Controller as launched with [MIP 24](../MIPS/mip-23) in order to avoid incentivising pools that receive a substantial amount of MTA per epoch but do not contribute effectively to the usage of the mStable protocol and add value by contributing to overall revenue.
```
Comment (dimsome) [inline MCCP/mccp-23.md:12]: link to mip 24

**Review by dimsome — CHANGES_REQUESTED** [review]
one tiny error :D

File: MCCP/mccp-23.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+mccp: 23
```
Comment (dimsome) [inline MCCP/mccp-23.md:2]: Something happened to the header no?

**Review by dimsome — APPROVED** [review]
LGTM

**Pull request merged**